lib/perf_hooks.js

```diff
@@ -341,19 +341,20 @@ class PerformanceObserver extends AsyncResource {
     if (typeof options !== 'object' || options === null) {
       throw new ERR_INVALID_ARG_TYPE('options', 'Object', options);
     }
-    if (!ArrayIsArray(options.entryTypes)) {
-      throw new ERR_INVALID_OPT_VALUE('entryTypes', options);
+    const { entryTypes } = options;
```

Comment:
this is to prevent accessing the value more than once (it could be a getter that doesn't always return the same thing)